content/contribute/contribute.md

```diff
@@ -142,4 +142,19 @@ more.
 ### Hiding pages
 To hide a page from the left-hand navigation use `tocHidden: true` in the front
 matter of the page. The docs website skips pages with `tocHidden:true` when
-building the menu.
\ No newline at end of file
+building the menu.
+
+### Changing page titles
+
+The version dropdown list that links the same page in different versions 
+together looks for pages with a matching title.
+
+If a page title changes use the front matter value `altTitle:` and a value of 
```

Comment:
Non-blocking suggestion: would `matchTitle` better convey the meaning of the field maybe?

Reply:
that is better! updated.